src/audiomixerboard.cpp

```diff
@@ -1183,7 +1183,7 @@ void CAudioMixerBoard::UpdateTitle()
     QString strEscServerName = strServerName;
     strEscServerName.replace ( "&", "&&" );
 
-    setTitle ( strTitlePrefix + tr ( "Personal Mix at: " ) + strEscServerName );
+    setTitle ( strTitlePrefix + tr ( "Personal Mix at: %1" ).arg ( strEscServerName ) );
```

Comment:
Why isn't `strTitlePrefix` a substitution, too?

Reply:
This would look a bit strange:
tr ( "%1Personal Mix at: %2") (note the missing space)

The RECORDING ACTIVE can be changed of course

Reply:
> This would look a bit strange:

That's not really much of an argument.  And the change at 1176 also wasn't what I meant.

Maybe the title of the PR is wrong and this isn't about using substitutions instead of concatenations but something else?

Reply:
Can you give a suggestion on what should be done? This PR is to make translations easier – as noted in the linked issue. If a change is not worth it – or even the whole PR, we can close the issue and PR

Reply:
Like you said:
```suggestion
    setTitle ( tr ( "%1Personal Mix at: %2" ).arg ( strTitlePrefix ).arg ( strEscServerName ) );
```
But only if using substitutions instead of concatenations is the goal.

Reply:
Personally, I don’t like this at all @BLumia @hoffie what do you think?